content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,67 @@
+
+We're introducing a new deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on separate clusters. This new deployment model allows a clear separation between mesh operators and mesh admins. With this new model, Istio mesh operators can run Istio control plane for mesh admins while mesh admins can still control the configuration of the control plane without worrying about installing or managing the Istio control plane. This model is transparent to mesh users.
+
+## Control Plane Cluster Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you have Istiod control plane installed in a single cluster like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+With this new deployment model in Istio 1.7, it is possible to run Istiod on a separate cluster (the `control plane cluster`) from services, as shown in the diagram below. The `control plane cluster` is owned by mesh operator while the mesh admin owns the cluster that has services deployed in the mesh. Mesh admin has no access to the `control plane cluster`. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
+
+{{< image width="100%"
+    link="single-cluster-central-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in a control plane cluster"
+    >}}
+
+Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the `control plane cluster`. The `primary data plane cluster` also serves as the Istio configuration cluster, in addition to run services. The `control plane cluster` reads Istio configurations from the `primary data plane cluster` and pushes them to all of the data plane clusters per the diagram below.
```

Comment:
Replace primary data plane cluster with config cluster, and say that the external control plane reads its configuration from the config cluster.

Reply:
Thanks Sven!  Updated PTAL